### PR TITLE
Fix external sharing loading script

### PR DESCRIPTION
**Pull request by skjnldsv**
What use had this file?

It doesn't seems to be used anywhere?

### CODE REVIEW

File: apps/federatedfilesharing/lib/AppInfo/Application.php [hunk 1]
```diff
@@ -98,14 +98,31 @@ function() use ($container) {
 				$server->getCloudFederationProviderManager()
 			);
 		});
+
+		// register events listeners
+		$eventDispatcher = $server->getEventDispatcher();
+		$manager = $server->getNotificationManager();
+		$federatedShareProvider = $this->getFederatedShareProvider();
```
Comment (nickvergessen) [inline apps/federatedfilesharing/lib/AppInfo/Application.php]: does this need to be executed outside of the listener?

Comment (skjnldsv) [reply apps/federatedfilesharing/lib/AppInfo/Application.php]: AH yes it seems to

Comment (skjnldsv) [reply apps/federatedfilesharing/lib/AppInfo/Application.php]: I reverted the protected https://github.com/nextcloud/server/blob/4bf5a23ec04539654b82170d9a4e6dabd21efa56/apps/provisioning_api/lib/Controller/UsersController.php#L438

**Comment by skjnldsv** [comment]
Ping ;)

**Review by rullzer — APPROVED** [review]
:elephant:

**Review by ChristophWurst — APPROVED** [review]
:elephant:

**Pull request merged**